### ISSUE TITLE
Intallation Ubuntu 22.04.1

### ISSUE THREAD

**Issue by AndyW999** [AcuGIS/OpenTileServer#82]
Hi
I tried the normal installation a couple of times but the rendering on a very fast machine I have used before was really slow and unusable.

So I thought I would try your Docker install but it failed with lots of font warnings and a load of errors.
Any ideas why?

Thanks

Andy

Loads of warnings for face-name then...

tile-1  | Mapnik LOG> 2024-09-27 09:53:50: warning: unable to find face-name 'Mukti Narrow Regular' in FontSet 'fontset-4'
tile-1  | Mapnik LOG> 2024-09-27 09:53:50: warning: unable to find face-name 'gargi Medium' in FontSet 'fontset-4'
tile-1  | Mapnik LOG> 2024-09-27 09:53:50: warning: unable to find face-name 'TSCu_Paranar Regular' in FontSet 'fontset-4'
tile-1  | Mapnik LOG> 2024-09-27 09:53:50: warning: unable to find face-name 'Tibetan Machine Uni Regular' in FontSet 'fontset-4'
tile-1  | Mapnik LOG> 2024-09-27 09:53:50: warning: unable to find face-name 'Mallige Normal' in FontSet 'fontset-4'
tile-1  | Mapnik LOG> 2024-09-27 09:53:50: warning: unable to find face-name 'Droid Sans Fallback Regular' in FontSet 'fontset-4'
tile-1  | Mapnik LOG> 2024-09-27 09:53:50: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-4'
pg-1    | 2024-09-27 09:53:50.747 UTC [34] ERROR:  relation "planet_osm_polygon" does not exist at character 36
pg-1    | 2024-09-27 09:53:50.747 UTC [34] STATEMENT:  SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE "way" IS NOT NULL LIMIT 1;
tile-1  | 
tile-1  | ** (process:89): ERROR **: 09:53:50.754: An error occurred while loading the map layer 'default': Postgis Plugin: ERROR:  relation "planet_osm_polygon" does not exist
tile-1  | LINE 1: SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE ...
tile-1  |                                            ^
tile-1  | in executeQuery Full sql was: 'SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE "way" IS NOT NULL LIMIT 1;'
tile-1  |   encountered during parsing of layer 'landuse_gen0' in Layer at line 252 of '/usr/local/share/maps/style/OSMBright/OSMBright.xml'
pg-1    | 2024-09-27 09:53:50.760 UTC [36] ERROR:  relation "planet_osm_polygon" does not exist at character 36
pg-1    | 2024-09-27 09:53:50.760 UTC [36] STATEMENT:  SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE "way" IS NOT NULL LIMIT 1;
pg-1    | 2024-09-27 09:53:50.763 UTC [34] ERROR:  relation "planet_osm_polygon" does not exist at character 36
pg-1    | 2024-09-27 09:53:50.763 UTC [34] STATEMENT:  SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE "way" IS NOT NULL LIMIT 1;
tile-1  | 
tile-1  | ** (process:89): ERROR **: 09:53:50.766: An error occurred while loading the map layer 'default': Postgis Plugin: ERROR:  relation "planet_osm_polygon" does not exist
tile-1  | LINE 1: SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE ...
tile-1  |                                            ^
tile-1  | in executeQuery Full sql was: 'SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE "way" IS NOT NULL LIMIT 1;'
tile-1  |   encountered during parsing of layer 'landuse_gen0' in Layer at line 252 of '/usr/local/share/maps/style/OSMBright/OSMBright.xml'
tile-1  | 
tile-1  | ** (process:89): ERROR **: 09:53:50.770: An error occurred while loading the map layer 'default': Postgis Plugin: ERROR:  relation "planet_osm_polygon" does not exist
tile-1  | LINE 1: SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE ...
tile-1  |                                            ^
tile-1  | in executeQuery Full sql was: 'SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE "way" IS NOT NULL LIMIT 1;'
tile-1  |   encountered during parsing of layer 'landuse_gen0' in Layer at line 252 of '/usr/local/share/maps/style/OSMBright/OSMBright.xml'
pg-1    | 2024-09-27 09:53:50.775 UTC [37] ERROR:  relation "planet_osm_polygon" does not exist at character 36
pg-1    | 2024-09-27 09:53:50.775 UTC [37] STATEMENT:  SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE "way" IS NOT NULL LIMIT 1;
tile-1  | 
tile-1  | ** (process:89): ERROR **: 09:53:50.781: An error occurred while loading the map layer 'default': Postgis Plugin: ERROR:  relation "planet_osm_polygon" does not exist
tile-1  | LINE 1: SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE ...
tile-1  |                                            ^
tile-1  | in executeQuery Full sql was: 'SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE "way" IS NOT NULL LIMIT 1;'
tile-1  |   encountered during parsing of layer 'landuse_gen0' in Layer at line 252 of '/usr/local/share/maps/style/OSMBright/OSMBright.xml'

**Comment by Bear4** [comment]
I have the same error:  ```
** (process:128): ERROR **: 08:49:21.451: An error occurred while loading the map layer 'default': Postgis Plugin: ERROR:  relation "planet_osm_polygon" does not exist
tile-1  | LINE 1: SELECT ST_SRID("way") AS srid FROM planet_osm_polygon WHERE ...

```

**Comment by AndyW999** [comment]
I have given up with this, it is not being maintained.

Both the normal installation and the Docker one does not work anymore.

I am using this now and it works OK - 

https://hub.docker.com/r/overv/openstreetmap-tile-server

**Comment by Bear4** [comment]
Yes, I've been using it for a few years now, but recently with the latest source it's having trouble rendering tiles...

**Comment by AndyW999** [comment]
Yes, same here.

Pity, it worked really well, so did their nominatim server when I used it but the one I referred to above works really well and even better for me as it will work in Windows ;)
